### PR TITLE
Bugfix/lattice 2342 linkingfeedbackcheck

### PR DESCRIPTION
**Pull request by you-leee**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/linking/graph/PostgresLinkingQueryService.kt [hunk 1]
```diff
@@ -293,35 +264,27 @@ class PostgresLinkingQueryService(private val hds: HikariDataSource, private val
...
-                    }
-                    val rs = ps.executeQuery()
-                    StatementHolder(connection, ps, rs)
-                },
-                Function<ResultSet, Pair<UUID, Set<UUID>>> {
-                    val linkingId = ResultSetAdapters.linkingId(it)
-                    val entityKeyIds = ResultSetAdapters.entityKeyIds(it)
-                    linkingId to entityKeyIds
-                })
+            normalEntitySetIds: Set<UUID>
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/linking/graph/PostgresLinkingQueryService.kt]: Are there corresponding changes in linker to include `normalEntitySetIds` in this call?

Comment (you-leee) [reply src/main/kotlin/com/openlattice/linking/graph/PostgresLinkingQueryService.kt]: Yes, totally forgot to push up the PR. Thanks!

Comment (you-leee) [reply src/main/kotlin/com/openlattice/linking/graph/PostgresLinkingQueryService.kt]: https://github.com/openlattice/linker/pull/39

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/linking/graph/PostgresLinkingQueryService.kt]: :D

**Comment by you-leee** [comment]
Waiting on this https://github.com/openlattice/conductor-client/pull/710 before merging anything

**Pull request merged**